### DISCUSSION TITLE
How to add code on after saved image, server.py

### DISCUSSION THREAD

**Discussion by Guiwoo** [Comfy-Org/ComfyUI#2800] [Q&A] [answered]
I would like to save image on other cloud platform, So I would like to add some code
but I've no idea where needs to attach the code. 
Wanna do something after save image !

**Comment by ltdrdata** [comment, marked as answer]
You can add your own node or you can hook this method.
https://github.com/comfyanonymous/ComfyUI/blob/7f89cb48bf8200254cde1306ba60d10ca019264d/nodes.py#L1433

**Reply by Guiwoo** [reply]
Thank you so much. it works !!
